esphome/components/online_image/online_image.h

```diff
@@ -83,8 +83,7 @@ class OnlineImage : public PollingComponent,
  protected:
   bool validate_url_(const std::string &url);
 
-  using Allocator = ExternalRAMAllocator<uint8_t>;
-  Allocator allocator_{Allocator::Flags::ALLOW_FAILURE};
+  RAMAllocator<uint8_t> allocator_;
```

Comment:
```suggestion
  RAMAllocator<uint8_t> allocator_{};
```
Although it will work without the braces, this explicitly calls the default constructor and also indicates to the reader that no further initialisation of this member is expected.